### PR TITLE
C#

### PR DESCRIPTION
**Pull request by jhollowayj**
Adding C# code and documentation.

### CODE REVIEW

File: examples/c#/CubicClient/Form1.Designer.cs [hunk 1]
```diff
@@ -0,0 +1,200 @@
+namespace CubicClient
+{
+    partial class Form1
```
Comment (mplooster) [inline examples/c#/CubicClient/Form1.Designer.cs]: A GUI? Fancy :)

File: examples/c#/CubicClient/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("CubicClient")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
```
Comment (mplooster) [inline examples/c#/CubicClient/Properties/AssemblyInfo.cs]: Should we include "Cobalt Speech and Language, Inc." as the company?

File: docs-src/content/getting-started/installation.md [hunk 3]
```diff
@@ -22,3 +22,13 @@ The Python SDK depends on Python >= 3.5. You may use pip to perform a system-wid
 pip install --upgrade pip
 pip install "git+https://github.com/cobaltspeech/sdk-cubic#egg=cobalt-cubic&subdirectory=grpc/py-cubic"
 ```
+
+### C#
+
+The C# SDK utilizes the [NuGet package manager](https://www.nuget.org).  The package is called `Cubic-SDK`, under the owners name of `CobaltSpeech`.
```
[the remark this answers is not in the document]
Comment (jhollowayj) [reply docs-src/content/getting-started/installation.md]: It looks like nuget supports mono, but I didn't try that. https://docs.microsoft.com/en-us/nuget/faqs/nuget-faq

File: docs-src/content/getting-started/recognize.md [hunk 4]
```diff
@@ -104,6 +104,47 @@ for result in resp.results:
 ```
 {{% /tab %}}
 
-{{%/tabs %}}
+{{% tab "C#" %}}
+``` c#
+// Initialize a gRPC connection
```
[the remark this answers is not in the document]
Comment (jhollowayj) [reply docs-src/content/getting-started/recognize.md]: That's not directly a C# issue, but I'll address it in another PR.

File: examples/c#/CubicClient/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (jhollowayj) [reply examples/c#/CubicClient/Properties/AssemblyInfo.cs]: Probably.  :D

File: grpc/csharp-cubic/.gitignore [hunk 5]
```diff
@@ -0,0 +1,349 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
+## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
```
Comment (renaissanceGeek) [inline grpc/csharp-cubic/.gitignore]: Wow, MS adds a lot of bloat! :smirk:

Comment (jhollowayj) [reply grpc/csharp-cubic/.gitignore]: I blindly took that from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore.  I hope that's ok.

Comment (renaissanceGeek) [reply grpc/csharp-cubic/.gitignore]: yes, my snark is directed at Microsoft, not you!

**Pull request merged**